cmd/kaspawallet/balance.go

```diff
@@ -22,10 +30,21 @@ func balance(conf *balanceConfig) error {
 		return err
 	}
 
-	fmt.Printf("Balance:\t\tKAS %f\n", float64(response.Available)/constants.SompiPerKaspa)
-	if response.Pending > 0 {
-		fmt.Printf("Pending balance:\tKAS %f\n", float64(response.Pending)/constants.SompiPerKaspa)
+	if conf.Verbose {
+		print("Address                                               ")
+	}
+	println("                        Available             Pending")
```

Comment:
I think displaying it as a table doesn't make sense when you have only one line, so I think it's better that the display will remain as before in non verbose mode.

Reply:
There are two lines. Second about pending balance.

Reply:
There _was_ a table before. With two lines and one column. Aligned by tab character.

The code will be more complex. The output will be ununified with verbose mode.

I am sure that now is better.

Reply:
Update:
Made it in one line

Reply:
```
Total balance, KAS    3249157.24327419
```

Reply:
It's very weird that the KAS comes before the actual number. What was bad with the way it was before?

Reply:
And generally speaking UX decisions shouldn't be affected by code complexity considerations when the difference in code complexity terms is so minor.

Reply:
It financial information systems is _very_ usual, when unit of measure is taken out column data. And there _is_ column data. Because when there is pending balance, output looks like
```
Total balance, KAS    3249157.24327419        500.00000000 (pending)
```
And with `--verbose`:
```
Address                                                                       Available             Pending
-----------------------------------------------------------------------------------------------------------
kaspa:qqcl3q8s8qdwxtm42fkjw6atu9wwafamv3tza26r4wvayh3jerkdv59z8l50d    3248804.12960945        500.00000000
kaspa:qzc34p2g0m04prt9akh3d7y0hqgxq07rl3yhskhjg6a9tx4vq38kq7mzwcylu        353.11366474
-----------------------------------------------------------------------------------------------------------
                                                 Total balance, KAS    3249157.24327419        500.00000000
```
It will be wierd when `KAS` will repeat after every number.

Now in `dev` branch `balance` output is:
```
Balance:		KAS 3249157.243274
```
There _is_ `KAS` before number. Why?

Reply:
I also not care about code complexity if UX requires it.
When I told about code complexity, I have in mind namely UX complexity. I made interface as unified as I can. Unified between output with and without `--verbose`. I tried to make as more common elements as I can, because more common elements means more simple UX (in general of cause).